tests/test_coastal_vulnerability.py

```diff
@@ -1112,6 +1112,46 @@ def test_aoi_multiple_features(self):
         n_points = layer.GetFeatureCount()
         self.assertTrue(n_points == 6)
 
+    def test_aoi_invalid_geometry(self):
+        """CV: test shore point creation with invalid geom in AOI."""
+        args = CoastalVulnerabilityTests.generate_base_args(self.workspace_dir)
+        aoi_path = os.path.join(self.workspace_dir, 'aoi.gpkg')
+        srs = osr.SpatialReference()
+        srs.ImportFromEPSG(26910)  # UTM Zone 10N
+        wkt = srs.ExportToWkt()
+        _ = make_vector_of_invalid_geoms(aoi_path)
+
+        # In this test, it won't matter if the landmass intersects the AOI.
+        landmass_path = os.path.join(
+            self.workspace_dir, 'landmass.geojson')
+        sampledata.create_vector_on_disk(

```

Comment:
I guess this wasn't raised as a merge conflict since this is a brand new test, but the use of `pygeoprocessing.testing.sampledata` is not compatible with `release/3.9`. 

I actually thought about this for a second as I did that CV PR. But I'm not sure I would do anything differently next time. Just as `sampledata` is not compatible with `release`, it's replacement `pygeoprocessing.shapely_geometry_to_vector` is not compatible with `master`.